### DISCUSSION TITLE
Increase total language count in top languages

### DISCUSSION THREAD

**Discussion by Santanu29** [anuraghazra/github-readme-stats#1619] [Ideas]
Can the total language count be increased from the current maximum of 10? I think it will be great to showcase all the languages a person has worked on.

**Comment by rickstaa** [comment]
@Santanu29 Currently, the language count is capped between 0-10. So using the `langs_count` variable as `20` doesn't work (see https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options). However, you are not the only one who thinks increasing the number of languages is useful (see https://github.com/anuraghazra/github-readme-stats/issues/1074#issuecomment-858408843 and https://github.com/anuraghazra/github-readme-stats/issues/909#issuecomment-788896431). Feel free to open [a feature request](https://github.com/anuraghazra/github-readme-stats/issues/new/choose).
